projects/lights_control/inc/lights_gpio.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
Please add module comments explaining a high-level overview of the purpose of this module and intended usage. This goes for all your modules.

Reply:
will do 👍

Reply:
You should take a look at our other modules (ex. `persist.h`) for an example of module-level documentation.